pkgs/tools/networking/curl/default.nix

```diff
@@ -89,23 +105,28 @@ stdenv.mkDerivation rec {
   '';
 
   configureFlags = [
+      # Build without manual
+      "--disable-manual"
```

Comment:
@Izorkin does this removes the man page for all curl variants?

Reply:
Command `man curl` works.

Reply:
`ls -lah ./result-man/share/man/man1`
```
того 56K
dr-xr-xr-x 2 root root    4 янв  1  1970 .
dr-xr-xr-x 3 root root    3 янв  1  1970 ..
-r--r--r-- 9 root root  52K янв  1  1970 curl.1.gz
-r--r--r-- 9 root root 1,8K янв  1  1970 curl-config.1.gz

```